omero_marshal/__init__.py

```diff
@@ -43,6 +45,22 @@ def get_decoder(t):
         return None
 
 
+def get_schema_version(version):
+    p = re.compile('^(\d+\.\d+\.\d+).*')
+    m = p.match(version)
+    if m is None:
+        raise Exception("Invalid OMERO version number")
+    v = StrictVersion(m.group(1))
+    if (v >= '5.1' and v < '5.3'):
```

Comment:
Pedantic, but it might be useful to explicitly say `5.1.0` and `5.3.0` here and in the `elif` below.